website/src/error-documentation.mjs

```diff
@@ -66,7 +66,7 @@ markdownParser.renderer.rules = {
     // Wrap BOM in a <span>.
     if (
       ["\u{feff}", "<mark>\u{feff}</mark>"].some((bom) =>
-        codeHTML.startsWith(bom)
+        codeHTML.includes(bom)
```

Comment:
Blocking: This is incorrect. U+FEFF *must* be at the the *beginning* to be considered a BOM. Otherwise, it is a joiner between characters. We don't want to show joiners as if they were a BOM.

Reply:
Okay, I fixed, I miss understand the concept of bom